src/components/shared/MembersList/MembersListItem.tsx

```diff
@@ -72,9 +74,15 @@ const MembersListItem = <U extends AnyUser = AnyUser>(props: Props<U>) => {
     [extraItemContent, user],
   );
 
-  const {
-    profile: { displayName, username },
-  } = userProfile;
+  const nativeToken = colony?.tokens?.items
+    .filter(notNull)
+    .find(
+      ({ token }) => token?.tokenAddress === colony.nativeToken.tokenAddress,
+    );
```

Comment:
I remember us having the native token available directly on the Colony object _(without having to filter for it)_.

Is that not the case anymore ?

Reply:
using correct nativeToken on the colony object now